docs/diagrams/LessonBookStudentBookModelClassDiagram.puml

```diff
@@ -9,7 +9,7 @@ skinparam classBackgroundColor MODEL_COLOR
         Class Student
 
         Class Name
-        Class Address
+        Class Student
```

Comment:
I think this should still be kept the same, like this:
```        Class Address```
as it actually stores an address

Reply:
I fixed the below one and forgot this 😢